app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx

```diff
@@ -102,7 +104,7 @@ export const MakeDecisionTabs = ({
   };
 
   return (
-    <div className={cx('make-decision-tabs')}>
+    <div className={cx('make-decision-tabs', { 'margin-top-10': isBulkOperation })}>
```

Comment:
May be better to use just `bulk-operation` class name here as we already know that it is a bulk operation and it may require additional styles in the future, not only `margin-top-10`.